CHANGELOG.md

```diff
@@ -5,6 +5,36 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to the versioning scheme outlined in the [README.md](README.md).
 
+## [2.05.0.0.0]
+
+This software update is a consensus changing release and the
+implementation of the proposed cost changes in SIP-012. This release's
+chainstate directory is compatible with chainstate directories from
+2.0.11.4.0, however, it is only compatible with chainstate directories
```

Comment:
Run-on sentence.  `However, it should only ...` should be a new sentence.

Also, can you drop a link here to the SIP 012 text?  Thanks

Reply:
Will fix the run on sentence.

As for a SIP-012 text link -- it hasn't merged yet, and I'm hesitant to link to a branch.